build-service.py

```diff
@@ -4,8 +4,9 @@
 KaaS Repo Build Script
 
 Usage:
-    build-service [-v | -vv] <repo-url> [--branch=<branch>]
-                  [--deploy-conf=<path/to/deploy.yml>] [--service-conf=<path/to/service.yml>]
+    build-service [-v | -vv] <repo-url> [--branch=<branch>] [--restart=<image_name>]
+                  [--delete=<image_name>] [--deploy-conf=<path/to/deploy.yml>]
+                  [--service-conf=<path/to/service.yml>]
```

Comment:
Restart and delete are exclusive, `|` marks them as such
```suggestion
    build-service [-v | -vv] <repo-url> [--branch=<branch>]
                  [--deploy-conf=<path/to/deploy.yml>] [--service-conf=<path/to/service.yml>]
                  [--restart=<image_name> | --delete=<image_name>]
```